docs/tutorials/goldenpath_series/gp_module_one.md

```diff
@@ -92,39 +155,39 @@ namespace HelloWorld
             GUILayout.Label("Mode: " + mode);
         }
 
-        static void SubmitNewPosition()
+static void SubmitNewPosition()
         {
             if (GUILayout.Button(NetworkManager.Singleton.IsServer ? "Move" : "Request Position Change"))
             {
-                var playerObject = NetworkManager.Singleton.SpawnManager.GetLocalPlayerObject();
-                var player = playerObject.GetComponent<HelloWorldPlayer>();
-                player.Move();
+                if (NetworkManager.Singleton.IsServer && !NetworkManager.Singleton.IsClient )
+                {
+                    foreach (ulong uid in NetworkManager.Singleton.ConnectedClientsIds)
+                        NetworkManager.Singleton.SpawnManager.GetPlayerNetworkObject(uid).GetComponent<HelloWorldPlayer>().Move();
+                }
+                else
+                {
+                    var playerObject = NetworkManager.Singleton.SpawnManager.GetLocalPlayerObject();
+                    var player = playerObject.GetComponent<HelloWorldPlayer>();
+                    player.Move();
+                }
             }
         }
     }
 }
 ```
 </details>
 
-5. **Save** your code in your text editor and return to Unity. Unity may take a moment to compile your changes.
-   :::note
-   You may see an error about `HelloWorldPlayer` not being found. This is okay; we will be addressing this error in a later section.
-   :::
 
-## Editor Modes to Golden Path (The HelloWorldManager Script)
+1. Add the `HelloWorldManager` script as a component.
```

Comment:
1. Add the `HelloWorldManager` script component to the `HelloWorldManager` `GameObject`.